src/index.html

```diff
@@ -4,9 +4,415 @@
+
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a class="header__link" href="/">
+          <picture>
+            <source srcset="/images/logo.png" media="(orientation: portrait)" >
+            <img src="/images/logo.png" alt="Logo" >
+          </picture>
+        </a>
+      </div>
+      <div class="header__block">
+        <nav class="header__navigation">
+          <ul class="header__menu">
+            <li class="header__navitem">
+              <a class="header__navlink is-active">Apple</a>
+            </li>
+
+            <li class="header__navitem">
+              <a class="header__navlink">Samsung</a>
+            </li>
+
+            <li class="header__navitem">
+              <a class="header__navlink">Smartphones</a>
+            </li>
+
+            <li class="header__navitem">
+              <a class="header__navlink" data-qa="nav-hover">Laptops & Computers</a>
+            </li>
+
+            <li class="header__navitem">
+              <a class="header__navlink">Gadgets</a>
+            </li>
+
+            <li class="header__navitem">
+              <a class="header__navlink">Tablets</a>
+            </li>
+
+            <li class="header__navitem">
+              <a class="header__navlink">Photo</a>
+            </li>
+
+            <li class="header__navitem">
+              <a class="header__navlink">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div class="card-grid">
+        <div class="card" data-qa="card">
+          <img src="./images/imac.jpeg" alt="Product image"
+          class="card__img">
+
+          <div class="card__content">
+            <p class="card__content card__content--title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+
+            <p class="card__content card__content--prcode">
+              Product code: 195434
+            </p>
+          </div>
+
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__rating card__rating--review">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price card__price--title">
+              Price:
+            </p>
+            <p class="card__price card__price--number">
+              $2,199
+            </p>
+          </div>
+
+          <div class="card__submit">
+            <a href="#" class="card__submit card__submit--button"
+            data-qa="card-hover" >
```

Comment:
Pay attention on code style. Fix all simila cases
In case tag has more than 2 attributes - each atribute should be from new line, should have 2 spaces, closed bracket should be on the same line as open.
```suggestion
            <a
              href="#"
              class="card__submit card__submit--button"
              data-qa="card-hover"
            >
```